ds-css-v2/styles/sass/cbp-theme-settings/_cbp-header.scss

```diff
@@ -183,17 +183,17 @@ $mobile-breakpoint: uswds-mobile-breakpoint.$theme-header-min-width - 1px;
 
       > span {
         display: inherit;
-        padding: $cbp-size-1 $cbp-size-2;
+        padding: 8px 16px; // $cbp-size-1 $cbp-size-2;
       }
     }
   }
 
   //Recently Visited Applications text
   .cbp-header-subtext {
     display: inline-block;
-    color: $cbp-accent-primary-light;
+    color: $cbp-blue-cool-30; // $cbp-accent-primary-light;
     font-style: italic;
-    margin: $cbp-size-2 $cbp-size-2 0 $cbp-size-2;
+    margin: 16px 16px 0 16px; // $cbp-size-2 $cbp-size-2 0 $cbp-size-2;
```

Comment:
@dannyk3941 Is there no pixel token for 16px?

Reply:
Resolved. The sizing tokens will be replaced once they are determined by the team.